conans/model/options.py

```diff
@@ -391,8 +391,9 @@ def __contains__(self, option):
     def loads(text):
         return PackageOptions(yaml.safe_load(text) or {})
 
-    def get_safe(self, field):
-        return self._data.get(field)
+    def get_safe(self, field, default=None):
+        result = self._data.get(field)
+        return default if result is None else result
```

Comment:
```suggestion
        return self._data.get(field, default)
```

It is a `dict`, isn't it?

Reply:
of course! 🤦‍♂